app/manifest.go

```diff
@@ -109,6 +111,8 @@ type ManifestKindVersion struct {
 	SelectableFields []string `json:"selectableFields,omitempty" yaml:"selectableFields,omitempty"`
 	// CustomRoutes is a map of of path patterns to custom routes for this version.
 	CustomRoutes map[string]spec3.PathProps `json:"customRoutes,omitempty" yaml:"customRoutes,omitempty"`
+	// GoKind is the association of this kind to the go resource.Kind type, which can be used for accessing the go type(s) and codec information
+	GoKind resource.Kind `json:"-" yaml:"-"`
```

Comment:
would it be easier/better to use the `ManifestGoTypeAssociator` util function for now and avoid modifying `ManifestKindVersion`?

Reply:
I think this might be the best approach, no need to make it more complicated right now with the addition of the AppManifest changes. The apiserver runner would just require the association function in its config so it can extract go types from the manifest.

Reply:
sounds good to me

Reply:
Alright, updated the PR to remove the `GoType` association in the manifest (and updated the description).